engineering/project-template/CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contribution Guidelines
```

Comment:
We're adding more and more files to the project root which aren't (hardly) related to the application/library itself. I wonder if moving some files to different directories could help to reduce the number of extra files; in this case, I'm wondering about what if we move into `.github` dir `CODE_OF_CONDUCT.md` and `CONTRIBUTING.md`

Reply:
I didn't know that `CONTRIBUTING.md` could be also stored in `docs` dir!!!
and it does https://help.github.com/articles/setting-guidelines-for-repository-contributors/

So yes... I'd prefer it too

Reply:
I will move this to a separate PR.